src/main/java/net/openhft/chronicle/core/util/StringUtils.java

```diff
@@ -330,6 +330,7 @@ public static byte[] extractBytes(@NotNull String s) {
         return getMemory().getObject(s, S_VALUE_OFFSET);
     }
 
+    @Deprecated(/* remove in x.26 */)
     public static void setCount(@NotNull StringBuilder sb, int count) {
         getMemory().writeInt(sb, SB_COUNT_OFFSET, count);
```

Comment:
should instead call `StringBuilder.setLength(count)`?

Reply:
It's not that straightforward. Bytes relies on this implementation. So it's gonna stay for now.
I don't intend to merge it yet.

Reply:
yep - I see it is an optimisation to avoid some setLength side-effects